ci/windows_dev_env/requirements.txt

```diff
@@ -0,0 +1,4 @@
+psutil
+boto3
+python-jenkins
```

Comment:
Where is this used?

Reply:
I think you added it, is it in the slave scripts?

Reply:
Well yeah it's used as part of the Autoconnect script. But since you are recommending this for users to install, it seems a bit out of place. We should handle infrastructure and MXNet related things separately if possible.

Reply:
we can remove it in the future,  it doesn't hurt there now, and I don't want additional scripts as of now. I don't think this is a big concern, even though you are right.